windows/eflow_deploy/check_eflow_vm.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Check EFLOW VM info
+- name: "Get EFLOW virtual machine power state"
+  include_tasks: ../utils/win_get_eflow_vm_state.yml
+- name: "Check EFLOW virtual machine power state"
+  ansible.builtin.assert:
+    that:
+      - win_eflow_vm_power_state == 'Running'
+    fail_msg: "Get EFLOW virtual machine power state is not 'Running': {{ win_eflow_vm_power_state }}"
+
+# Check EFLOW VM OS info
+- name: "Get EFLOW virtual machine OS info"
+  include_tasks: ../utils/win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "Invoke-EflowVmCommand -command 'cat /etc/os-release'"
+- name: "Check EFLOW virtual machine OS info"
+  ansible.builtin.assert:
+    that:
+      - win_powershell_cmd_output.stdout is defined
+      - win_powershell_cmd_output.stdout
```

Comment:
This line can be removed as if it is empty, the following checking would also fail.